src/tokens.scss

```diff
@@ -0,0 +1,11 @@
+:host {
+  --ds-auro-button-text-color: var(--ds-color-text-primary-inverse, $ds-color-text-primary-inverse);
+  --ds-auro-button-container-color: var(--ds-color-ui-default-default, $ds-color-ui-default-default);
+  --ds-auro-button-container-image: var(--ds-color-ui-default-default, $ds-color-ui-default-default);
```

Comment:
I feel like this is a sort of confusing name. In another repo, I used something along the lines of `button-container-gradient-one`